internal/controllers/common/resource_definitions/resource_definitions.go

```diff
@@ -1763,7 +1763,8 @@ func NewStorageContainer(cr *model.CryostatInstance, imageTag string, tls *TLSCo
 	args := []string{}
 	if tls != nil {
 		args = append(args,
-			fmt.Sprintf("-s3.port.https=%d", constants.StoragePortTLS),
+			fmt.Sprintf("-s3.port=%d", constants.StoragePort),
+			"-s3.port.https=0",
```

Comment:
This might be a bit confusing to come across later down the road. Maybe we could either remove the explicit `s3.port.https` option, or give a comment that explains that the `s3.port` value will be used in this case?